src/components/util/code-quarkus-url.js

```diff
@@ -0,0 +1,31 @@
+import { default as parse } from "mvn-artifact-name-parser"
+
+const codeQuarkusUrl = ({ artifact, unlisted, platforms, streams }) => {
+  // Do some light pre-checking so we don't have to deal with catching
+  if (artifact && artifact.includes(":")) {
+    if (!unlisted) {
+      const coordinates = parse(artifact)
+      const artifactId = coordinates.artifactId
+      if (
+        platforms &&
+        platforms.includes("quarkus-bom-quarkus-platform-descriptor")
+      ) {
+        const trimmedArtifactId = artifactId.replace(/^quarkus-/, "")
```

Comment:
that might help: https://github.com/quarkusio/code.quarkus.io/blob/main/library/components/api/quarkus-project-utils.ts#L108-L110

id is `groupId:artifactId`

Reply:
ah, no sorry, it's not needed in your case, mine is working when it's already a shortId and without checking for platform, so in your case, yours is good to, .... nevermind my comment

Reply:
ok! I was just looking at the first comment and I thought your logic was a bit tidier than mine. but if you think it's ok as is I won't touch anything :)